movies_import/utils.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import requests
+from django.contrib import messages
+from django.core.exceptions import ValidationError
+from zipfile import ZipFile
+from django.db import IntegrityError
+from cinema.settings import MEDIA_ROOT
+import csv
+from pathlib import Path
+from django.core.files import File
+from movies.models import Movie
+
+
+def validate_zipfile(zip_file):
+    ext = os.path.splitext(zip_file.name)[1]
+    valid_ext = ['.zip', '.rar']
+    if not ext.lower() in valid_ext:
+        raise ValidationError('Unsupported file extension')
+
+
+def get_movies_form_zip(file, path):
+    zip_file = ZipFile(file)
+    files = zip_file.namelist()
+    for f in files:
+        if f.endswith('.csv'):
+            zip_file.extractall(path=path)
+            return f
+
+
+def import_movies_from_zip(file, name):
+    path = f'{MEDIA_ROOT}'
+    csv_file = get_movies_form_zip(file, path)
+    with open(f"{path}\{csv_file}", 'r') as f:
+        data = csv.DictReader(f)
+        added_movies = []
+        system_list = Movie.objects.values_list('pk', flat=True)
+        for m in data:
```

Comment:
Try to use suggestive names for variables. `m` doesn't ring any bell for me, I have to dive into the functionality to understand. Make it as literal as possible. You can even write something like `for movie_data in csv_data` or something like that.